src/modules/solvers/QiskitQAOA.py

```diff
@@ -109,8 +114,8 @@ def get_parameter_options(self) -> dict:
                 "description": "How many shots do you need?"
             },
             "iterations": {  # number measurements to make on circuit
-                "values": [10, 20, 50, 75],
-                "description": "How many iterations do you need?"
+                "values": [1, 5, 10, 20, 50, 75],
+                "description": "How many iterations do you need? Warning: When using the IBM Eagle Device you should only choose a lower number of iterations, since a high number would lead to a waiting time that could take up to mulitple days!"
```

Comment:
Maybe there is a way to only raise the warning if the IBM Eagle device is selected.